Web/starter_project_vanguard/components/teams/MemberCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { Member } from '@/types/teams'
+import Image from 'next/image'
+import Link from 'next/link'
+import React from 'react'
+import { FaFacebook, FaInstagram, FaLinkedin } from 'react-icons/fa'
+import { IconContext } from 'react-icons/lib'
+
+interface MemberCardProps {
+  member: Member
+}
+const MemberCard: React.FC<MemberCardProps> = ({ member }) => {
+  let social_icon_map = new Map<string, JSX.Element>()
+  social_icon_map.set('facebook', <FaFacebook />)
+  social_icon_map.set('linkedin', <FaLinkedin />)
+  social_icon_map.set('instagram', <FaInstagram />)
```

Comment:
1. let's stick to camelCase for variable
2. it might be cleaner to just build an object (key value pair)
```
const socialMediaIcons = {
  facebook: <FaFacebook />,
  linkedin: <FaLinkedin />
  instagram: <FaInstagram />
  default: [some default icon to use when it is not in here, could be FaWeb or sth]
}
```

and when using it you can do `socialMediaIcons[link.social] || socialMediaIcons.default` to make it use the default when undefined

Reply:
and for general knowledge, you don't need to use `let` as long as you are not reassigning the variable. it should stay as `const`